From 0efbf32e1bcd1cfe046df072cbbc32182cca29a7 Mon Sep 17 00:00:00 2001
From: Gatis Paeglis <gatis.paeglis@theqtcompany.com>
Date: Thu, 12 May 2016 11:44:35 +0200
Subject: [PATCH] Fix build for HAVE_LIBSOUP_CLIENT_CERTS

---
 src/libostree/ostree-fetcher.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/libostree/ostree-fetcher.c b/src/libostree/ostree-fetcher.c
index d7915ba..d54b7c7 100644
--- a/src/libostree/ostree-fetcher.c
+++ b/src/libostree/ostree-fetcher.c
@@ -273,6 +273,7 @@ session_thread_set_proxy_cb (ThreadClosure *thread_closure,
                 proxy_uri, NULL);
 }
 
+#ifdef HAVE_LIBSOUP_CLIENT_CERTS
 static void
 session_thread_set_tls_interaction_cb (ThreadClosure *thread_closure,
                                        gpointer data)
@@ -288,6 +289,7 @@ session_thread_set_tls_interaction_cb (ThreadClosure *thread_closure,
                 SOUP_SESSION_TLS_INTERACTION,
                 interaction, NULL);
 }
+#endif
 
 static void
 session_thread_set_tls_database_cb (ThreadClosure *thread_closure,
-- 
2.7.4

